tests/README.md

```diff
@@ -86,27 +49,79 @@ service (or your `QGIS_PGTEST_DB` connection string) uses a connection
 method which allows passing username/password pair to connect (and
 which allows creating users)
 
-# Write tests
+Some tests require a specific PostgreSQL server configuration
+(installation of auth certificates) so the most convenient way
+to bring up such server would be to:
 
-Instructions about writing tests for the processing framework
-can be found in a [separate README file](../python/plugins/processing/tests/README.md):
+    GH_WORKSPACE=$PWD \
```

Comment:
I think this is competing with another open pr of you?

Reply:
Yes but that PR is failing so better not wait for it before improving the README. Eventually I'll change the README in the associated branch when it passes (it's #47060 for the record)